src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Compilation.targets

```diff
@@ -111,6 +111,7 @@ Copyright (c) .NET Foundation. All rights reserved.
       <UseSharedCompilation>true</UseSharedCompilation>
     </PropertyGroup>
 
+    <!-- TODO: does this need to be updated for 5.0? -->
```

Comment:
Do we need to update this calculation of `MaxSupportedLangVersion`?

Reply:
I'd remove the comment. This target is kept in sync with Roslyn's compilation targets. We'll update it once they've updated theirs.